package.json

```diff
@@ -68,8 +68,10 @@
   ],
   "dependencies": {
     "cornerstone-math": "^0.1.7",
+    "dicomweb-client": "^0.4.1",
```

Comment:
@swederik  I needed to add this dependency, because it is required in another part of ohif-core. Is that correct?